environment.yml

```diff
@@ -7,13 +7,12 @@ dependencies:
   - numpy=1.21
   - astropy~=4.2
   - pip
-  - ctapipe~=0.11
-  - gammapy=0.18
+  - ctapipe=0.12
+  - gammapy=0.19
```

Comment:
Should we also get the upcoming gammapy 1.0? Will there be a significant change in gammapy that will affect the few functions used in lstchain?

Reply:
Since gammapy 1.0 is not released yet and won't be for some time, I don't get the point you are making.

Reply:
I assumed that this week's coding sprint would push for the next release. I guess, we can change this later then.

Reply:
The time frame was 2-3 month according to regis